pkg/apis/garden/types.go

```diff
@@ -1011,6 +1018,10 @@ type Local struct {
 	Networks LocalNetworks
 	// Endpoint of the local service.
 	Endpoint string
+	// MachineImage holds information about the machine image to use for all workers.
+	// It will default to the first image stated in the referenced CloudProfile if no
```

Comment:
I receive panic when I remove the `machineImage` part from the local shoot spec. You could remove the `defaulting` part from the doc or add the defaulting logic?

Reply:
Fixed: default set at shoot validation plugin, as suggested by @rfranzke